src/Build/BackEnd/Components/Logging/ProjectLoggingContext.cs

```diff
@@ -80,8 +81,8 @@ private ProjectLoggingContext(NodeLoggingContext nodeLoggingContext, int submiss
             ProjectPropertyInstanceEnumeratorProxy properties = null;
             ProjectItemInstanceEnumeratorProxy items = null;
 
-            IEnumerable<ProjectPropertyInstance> projectPropertiesEnumerator = projectProperties == null ? Array.Empty<ProjectPropertyInstance>() : null;
-            IEnumerable<ProjectItemInstance> projectItemsEnumerator = projectItems == null ? Array.Empty<ProjectItemInstance>() : null;
+            IEnumerable<ProjectPropertyInstance> projectPropertiesEnumerator = projectProperties == null ? Enumerable.Empty<ProjectPropertyInstance>() : null;
+            IEnumerable<ProjectItemInstance> projectItemsEnumerator = projectItems == null ? Enumerable.Empty<ProjectItemInstance>() : null;
```

Comment:
Better yet, inline these. It looks like there are several opportunities for simplifying this constructor.

Reply:
Apologies, I didn't get to this before it was merged. Looks like it's been done in #7117 though.

Reply:
Yes! No worries; I saw that and marked this merge-when-branch-open.